options/cf_options.cc

```diff
@@ -408,6 +408,14 @@ static std::unordered_map<std::string, OptionTypeInfo>
          {offsetof(struct MutableCFOptions, blob_compression_type),
           OptionType::kCompressionType, OptionVerificationType::kNormal,
           OptionTypeFlags::kMutable}},
+        {"enable_blob_garbage_collection",
+         {offsetof(struct MutableCFOptions, enable_blob_garbage_collection),
```

Comment:
First, would it be better to create a struct that held all of the Blob options in it, rather than adding them one at a time to the various options (ColumnFamilyOptions, MutableCFOptions).  

Second, "blob_garbage_collection_enabled" would be a better name.  That way all of the "blob" options start with "blob" and will be grouped together alphabetically.